src/Car.java

```diff
@@ -34,17 +35,23 @@ public Car(String model, String owner) {
 
     public double valueAfter(int years, double milesDriven) {
         //value * depreciationMilesFactor * milesDriven
-        double depreciatedMilesValue = value*depreciationMilesFactor*milesDriven;
+        double depreciatedMilesValue = value * depreciationMilesFactor * milesDriven;
         //value x depreciationTimeFactor * years
-        double depreciatedTimeValue = value*depreciationTimeFactor*years;
+        double depreciatedTimeValue = value * depreciationTimeFactor * years;
         //subtract both top from value initial.
         double depreciatedValue = value - depreciatedMilesValue - depreciatedTimeValue;
         return depreciatedValue;
     }
 
-    public double costOfGas(float milesDriven) {
+    public double costOfGas(double milesDriven) {
         //costOfGas = milesDriven / milesPerGallon * gasPrice
-        double overallGasCost = milesDriven/milesPerGallon*2;
+        double overallGasCost = milesDriven / milesPerGallon * 2;
         return overallGasCost;
     }
-}
\ No newline at end of file
+
+    public double lifetimeGasUsed(double milesDriven, double milesPerGallon) {
```

Comment:
this function receives double milesPerGallon but the class Car already has a field called milesPerGallon.

remove double milesPerGallon from function signature